### PR TITLE
Add progress bar while preflight checks are running

### PR DESCRIPTION
**Pull request by GraysonNull**
Screen Shot 2021-03-23 at 10 50 50 AM

### CODE REVIEW

File: web/src/components/troubleshoot/PreflightsProgress.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React from "react"
+import Loader from "../shared/Loader";
+import "@src/scss/components/AirgapUploadProgress.scss";
+import { getReadableCollectorName } from "../../utilities/utilities";
+
+export default class PreflightProgress extends React.Component {
```
Comment (jgruica) [inline web/src/components/troubleshoot/PreflightsProgress.jsx:6]: mind changing this to functional component because it's stateless?

File: web/src/utilities/utilities.js [hunk 2]
```diff
@@ -55,6 +55,22 @@ export function getApplicationType(watch) {
...
+    "cluster-resources": "Gathering available resources in cluster",
+    "mysql": "Gathering information about MySQL",
+    "postgres": "Gathering information about PostgreSQL",
+    "redis": "Gathering information about Redis"
+  }
+  const statusToReturn = namesObj[name];
+  if (statusToReturn) {
+    return statusToReturn
+  } else {
+    return `Gathering details about the cluster`;
```
Comment (jgruica) [inline web/src/utilities/utilities.js:70]: why `? can this be "Gathering details about the cluster"?

File: web/src/components/troubleshoot/PreflightsProgress.jsx (hunk 1 above)
Comment (GraysonNull) [reply web/src/components/troubleshoot/PreflightsProgress.jsx:6]: Yeah sure thing.

File: web/src/components/gitops/GitOpsRepoDetails.jsx [hunk 3]
```diff
@@ -206,28 +206,6 @@ class GitOpsRepoDetails extends React.Component {
                   </label>
                 </div>
               </div>
-              <div className="BoxedCheckbox-wrapper flex1 u-textAlign--left u-marginLeft--10">
```
Comment (jgruica) [inline web/src/components/gitops/GitOpsRepoDetails.jsx:209]: why is this deleted?

**Review by jgruica — APPROVED** [review]

File: web/src/utilities/utilities.js (hunk 2 above)
Comment (GraysonNull) [reply web/src/utilities/utilities.js:70]: I'm not sure I follow.

Comment (jgruica) [reply web/src/utilities/utilities.js:70]: does it need to be as backtick? or it can be inside quotation mark?

Comment (GraysonNull) [reply web/src/utilities/utilities.js:70]: Oh I see. No can be quotation marks.
